dada_tests/interpret/ops_uint.dada

```diff
@@ -0,0 +1,66 @@
+async fn main() {
+    var uint_max = 18446744073709551615
+
+    if 1 + 1 == 2 {
+        print("1 + 1 == 2").await
+    }
+
+    # wrapping overflow
+    if uint_max + 2 == 1 {
```

Comment:
we should decide what to do here =) bah, the annoying things. Maybe return an Err for now?